src/gpu/amd/sycl_hip_utils.hpp

```diff
@@ -211,6 +211,15 @@ inline status_t create_and_set_tensor_descriptor(
     return status::success;
 }
 
+static status_t create_and_set_tensor_descriptor_ex(
+        miopenTensorDescriptor_t *tensor_desc, miopenTensorFormat_t format,
```

Comment:
miopenTensorFormat_t is supposed to be miopenTensorLayout_t, we will change it. 
Regarding create_and_set_tensor_descriptor_ex, as you pointed out this function is not necessary for MIOpen  we will remove this and use this create_and_set_tensor_descriptor.

Reply:
> miopenTensorFormat_t is supposed to be miopenTensorLayout_t, we will change it.

I think we should refrain from using `miopenTensorLayout_t` because it was introduced only 2 months ago and there is a chance that this API will be reverted before the next MIOpen release (see discussion https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1532).